packages/core/test/controls/numericInputTests.tsx

```diff
@@ -0,0 +1,736 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import { expect } from "chai";
+import { mount, ReactWrapper } from "enzyme";
+import * as React from "react";
+
+import * as Errors from "../../src/common/errors";
+import { Classes, Keys, NumericInput, Position } from "../../src/index";
+
+describe("<NumericInput>", () => {
+
+    describe("Defaults", () => {
+
+        it("renders the buttons on the right by default", () => {
+            const component = mount(<NumericInput />);
+            const inputGroup = component.childAt(0);
+            expect(inputGroup.name()).to.equal("Blueprint.InputGroup");
```

Comment:
what's wrong with `expect(component.find(InputGroup)).to.exist` ? (although that code is not correct but you get the idea. enzyme has some powerful finders.)